inc/Engine/Preload/Controller/PreloadUrl.php

```diff
@@ -171,7 +171,7 @@ protected function get_mobile_user_agent_prefix() {
 			return $prefix;
 		}
 
-		return $new_prefix;
+		return 'WP Rocket ' . $new_prefix;
```

Comment:
```suggestion
		return 'WP Rocket/Preload ' . $new_prefix;
```

Should we do that so we know it's the preload?